### ISSUE TITLE
Running `upgrade` un-pins dependencies

### ISSUE THREAD

**Issue by tzachbon** [wixplosives/pleb#520]
When running `upgrade` it also "un-pins" a version even though it wasn't upgraded.

![image](https://user-images.githubusercontent.com/45866571/208796243-99a65108-d816-4fc5-9f8f-a47fae7df648.png)

It's a little unexpected. Should it keep the request status (minor, patch, pinned)?

**Comment by AviVahl** [comment]
It's actually on purpose. Whether it is correct to do so, could be argued... Especially in a semver ecosystem and when making use of lock files.

You can tell pleb to ignore specific dependencies using `pleb.config.js/mjs`: https://github.com/wixplosives/pleb#configuration-file

**Comment by tzachbon** [comment]
> It's actually on purpose. Whether it is correct to do so, could be argued... Especially in a semver ecosystem and when making use of lock files.
> 
> You can tell pleb to ignore specific dependencies using `pleb.config.js/mjs`: [wixplosives/pleb#configuration-file](https://github.com/wixplosives/pleb#configuration-file)

I agree with you, but there could be a case where an installed library would like a specific version, and we know it can't control its user's lock file so the author has to pin the version in the package json file. In that case, the library author can't use pleb to upgrade it and would have to do it manually.
